install_config/storage_examples/direct_storage.adoc

```diff
@@ -0,0 +1,481 @@
+= Direct access to storage using two methods
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+In OpenShift a Persistent Volume Claim (PVC) will bind to a Persistent Volume (PV) based on matching the
```

Comment:
Consider adding a note that both of the methods you describe may be not be directly supported in future releases since 1) defining the plug-in directly in the pod spec will,  by default, not be allowed (need SCC setting to allow it), and 2) there is pressure to remove support for ClaimRef in a PV spec.